src/backend/web/routes/users.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+
+const { logger } = require('../../utils/logger');
+const feeds = require('../../data/feeds');
+const { getFeeds } = require('../../utils/storage');
+
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+  // this route will return all users.
+  /* [{id: "some-user-id", url: "/users/some-user-id", ...] */
+});
+
+router.get('/:id', async (req, res) => {
+  // this route will return one user, and there list of feeds, this can be utilized in two places. One our search function, and
```

Comment:
>and there list of feeds

(For future reference/reminder: as per discussion in today's class, implementing this will require that we first ensure that feed information stored in Redis includes the user's ID.)

Reply:
Yes - so that's the part I think I'll need help with is, figuring out how the hashing function works. 

I believe right now we hash the URL for the feed, but going forward we would be hashing the username if I'm not mistaken?

Reply:
@humphd should we hash usernames?

Reply:
Personally, I'm not sure we need to expose Users to the frontend at all.  We want it to exist internally in Redis, so we can match logged in users with their feeds; but we don't need to list all our users, so we can probably get rid of `/users` as a route.

Reply:
I have removed the /users route, so it would be /users/:id only.